pkg/proc/native/support_sentinel_linux.go

```diff
@@ -1,4 +1,4 @@
-//go:build linux && !amd64 && !arm64 && !386 && !(ppc64le && exp.linuxppc64le)
+//go:build linux && !amd64 && !arm64 && !386 && !(ppc64le && exp.linuxppc64le) && !riscv64
```

Comment:
If we don't have a build agent for riscv64 this needs to be guarded with a build flag like ppc64le.

Reply:
Added.

Reply:
That could be done. The instructions to add a teamcity agent are here: https://www.jetbrains.com/help/teamcity/install-teamcity-agent.html.

Reply:
Thanks, we could provide riscv hardware for this, then how could we add this in delve's teamcity just like ppc64le? Also, does this ci machine need to be controled by delve dev team directly or we could host it in a third-party organization?